eng/CodeAnalysis.src.globalconfig

```diff
@@ -468,6 +468,9 @@ dotnet_diagnostic.CA1858.severity = warning
 # CA1859: Use concrete types when possible for improved performance
 dotnet_diagnostic.CA1859.severity = warning
```

Comment:
Do things get out of hand if you turn on analysis of internal stuff too?

Reply:
Yes, the default is private and you can opt-in to internal and public too.

Support for internal and public is in a PR that is about to be merged: https://github.com/dotnet/roslyn-analyzers/pull/6421

Reply:
Yes, I tried with internal a lot flagged that looks valid, now https://github.com/dotnet/roslyn-analyzers/pull/6421 is merged, probably better wait until it is ready for consumption for the repo before enable the internal API surface